Gemfile.lock

```diff
@@ -337,5 +337,8 @@ DEPENDENCIES
   webmock (~> 3.8.3)
   yard
 
+RUBY VERSION
+   ruby 2.6.6p146
+
```

Comment:
I don't think we need the Ruby version here, as there's a `.ruby-version` file with this information in it in the repo already.
```suggestion
```